### DISCUSSION TITLE
Frigate cache in RAM memory recommended

### DISCUSSION THREAD

**Discussion by xekil** [tteck/Proxmox#3599] [General]
Hello,

Following a question about [SSD wear with Frigate](https://github.com/blakeblackshear/frigate/discussions/13359), I just learned that it was recommended to move the Frigate cache into RAM, but this is not documented in the case of installing Frigate in an LXC container without Docker.

>https://docs.frigate.video/frigate/installation/#storage
**Storage:**
 /tmp/cache: Cache location for recording segments. Initial recordings are written here before being checked and converted to mp4 and moved to the recordings folder. Segments generated via the clip.mp4 endpoints are also concatenated and processed here. It is recommended to use a [tmpfs](https://docs.docker.com/storage/tmpfs/) mount for this.

Have any of you already done this?
And if so, can you detail how to do it?

This seems useful in order not to prematurely wear out the SSD containing Proxmox and the LXC Container.

Thanks

**Hardware:**

- HP EliteDesk 800 G3
- Proxmox 8.2.4
- i5 7600T 2.80GHz
- 32 GB RAM
- Container LXC Frigate 0.14.0, etc...
- SSD PNY 500Go M280CS2230-500-RB
- HDD USB WD 2To (media folder for Frigate)
- CORAL USB
- NAS storage (for Backup containers LXC)

**Comment by xekil** [comment]
No one uses this recommendation with Frigate LXC ?

**Comment by remz1337** [comment]
This is already taken care of by this line.
https://github.com/tteck/Proxmox/blob/4ead590cf46585069d6b71ec34ebdb4c062cdc70/install/frigate-install.sh#L104

**Comment by xekil** [comment]
good new thank you
